cli/tests/test_commit_command.rs

```diff
@@ -73,6 +73,28 @@ fn test_commit_with_editor() {
     "###);
 }
 
+#[test]
+#[cfg(windows)]
+fn test_commit_with_editor_avoids_unc() {
+    let mut test_env = TestEnvironment::default();
+    test_env.jj_cmd_ok(test_env.env_root(), &["git", "init", "repo"]);
+    let workspace_path = test_env.env_root().join("repo");
+    let edit_script = test_env.set_up_fake_editor();
+
+    std::fs::write(edit_script, ["dumppath path"].join("\0")).unwrap();
+    test_env.jj_cmd_ok(&workspace_path, &["commit"]);
+
+    let editor_path = std::fs::read_to_string(test_env.env_root().join("path")).unwrap();
```

Comment:
nit: is it simpler to wrap this in `Path::new()`? then you don't need to call `.to_str().unwrap()` below, i think. Same in the other tests

Reply:
Done - also applied in #3995.